src/transformStateWithClones.js

```diff
@@ -1,13 +1,32 @@
 'use strict';
 
 /**
- * @param {Object} state
- * @param {Object[]} actions
+ * @param {objetoect} state
+ * @param {objetoect[]} actions
  *
- * @return {Object[]}
+ * @return {objetoect[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let objeto = { ...state };
+  const novo = [];
```

Comment:
The variable name `novo` is not descriptive of what it holds. Consider renaming it to `statesHistory` or `stateClones` to better reflect its purpose.